HHS/OPRE-OPS
# use consistent schema for change requests response

## What changed

* fix: `has_status_change`, `has_budget_change` were missing from the change requests list response
* use the same schema for `/change-requests/` that is used with the embedded change requests for agreements/BLIs (instead of to_dict)

## Issues

#2307 

## How to test

* run the tests (checks for  `has_status_change`, `has_budget_change` were added to existing tests in  `test_workflows.py`)
* you can also manually verify that the missing properties are included in the response for `/change-requests/`


## Definition of Done Checklist
